### ISSUE TITLE
Fine tuning

### ISSUE THREAD

**Issue by laugam** [hzwer/Practical-RIFE#19]
Hi, thank you for making your code available. The latest model release (v4.6) works really well for my project. 
I would like to fine tune the model on my dataset.
Do you plan to make the script for the fine tuning available any time soon? 
Thank you.

**Comment by laugam** [comment]
Thank you for your reply. 
The train and evaluate functions that you defined in the file train.py don't seem to be compatible with the new files RIFE_HDv3.py and IFNet_HDv3.py, for instance the loss functions are different. 
Do you have any suggestions on how to modify those functions?

Many thanks.

**Comment by xsp1** [comment]
Hello, I am also trying to retrain or fine-tune the v4.6 model on my own data set recently. I also found that RIFE_HDv3.py and IFNet_HDv3.py are different from the original version. I modified a lot of codes, such as removing After using loss_cons, I found that the effect is very poor after successful training. Did you fine-tune successfully? How did you solve these problems? Thanks!
